bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/CTabFolder.java

```diff
@@ -3694,6 +3707,25 @@ public void setUnselectedImageVisible(boolean visible) {
 	showUnselectedImage = visible;
 	updateFolder(REDRAW);
 }
+/**
+ * Specify whether the image appears on selected tabs.
+ *
+ * @param visible <code>true</code> makes the image appear
+ *
+ * @exception SWTException <ul>
+ *    <li>ERROR_WIDGET_DISPOSED - if the receiver has been disposed</li>
+ *    <li>ERROR_THREAD_INVALID_ACCESS - if not called from the thread that created the receiver</li>
+ * </ul>
+ *
+ * @since 3.124
```

Comment:
I think we need a version bump to 3.125 because of API extension here, don't we? 3.124.0 and 3.124.100 won't contain this API.

Reply:
Yes. We will bump it in a new change.